src/main/java/io/strimzi/kafka/bridge/http/HttpMode.java

```diff
@@ -0,0 +1,32 @@
+package io.strimzi.kafka.bridge.http;
```

Comment:
This enum is not needed as it exists for AMQP bridge (`AmqpMode`).
The AMQP mode is needed because the AMQP bridge can work in two ways : alone so it acts like a server accepting AMQP connections or with the Qpid Dispatch Router; in the latter case the AMQP bridge connect to the router acting in client mode and the router listen for connections from AMQP clients forwarding traffic to the AMQP bridge (but it's something you don't need to know for our scope ... just fyi :-))